startup_heroku.sh

```diff
@@ -29,5 +30,6 @@ export SMART_ANSWER_TO_OPEN="marriage-abroad"
 open $HEROKU_URL$SMART_ANSWER_TO_OPEN
 
 echo
+echo "*Set ERRBIT_API_KEY and ERRBIT_HOST manually to enable error reporting*"
```

Comment:
Can we print some information about where to find the `ERRBIT_API_KEY` and `ERRBIT_HOST`? I wouldn't know where to find those at the moment.